inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/textureEditorComponent.tsx

```diff
@@ -313,7 +315,7 @@ export class TextureEditorComponent extends React.Component<ITextureEditorCompon
                 pickerOpen={this.state.pickerOpen}
                 setPickerOpen={this.setPickerOpen}
                 pickerRef={this._pickerRef}
-                hasAlpha={this.props.texture.hasAlpha}
+                hasAlpha={this.props.texture.textureFormat === -1 || this.props.texture.textureFormat === Constants.TEXTUREFORMAT_RGBA}
```

Comment:
-1 means?

Reply:
Its the default value assigned. We take it as ok to be able to use the alpha.

Reply:
ok sounds good! thanks!